### PR TITLE
Add a CHANGELOG and update the README

### PR DESCRIPTION
**Pull request by geowarin**
Hi guys,

I just wanted to give a hand with #84.
I realise it is not perfect so feel free to use it as a base or make a better PR, I won't be offended :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,15 +154,27 @@ piece of state must be named `routing`** (unless you provide a custom
 
 ### `UPDATE_PATH`
 
-An action type that you can listen for in your reducers.
+An action type that you can listen for in your reducers to be notified of
+route updates.
 
-### `updatePath(path, noRouterUpdate)`
+### `pushPath(path, state = {}, avoidRouterUpdate = false)`
```
Comment (kjbekkelund) [inline README.md]: This does not match the api: https://github.com/jlongster/redux-simple-router/blob/3545a3e7fea0d8629180c774e16825d0d4ce547c/src/index.js#L23 (e.g. `state` does not have a default value)

I actually think you can just copy the line from the code.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Change Log
```
Comment (kjbekkelund) [inline CHANGELOG.md]: "Changelog" in one word

File: README.md [hunk 3]
```diff
@@ -154,15 +154,27 @@ piece of state must be named `routing`** (unless you provide a custom
...
-An action creator that you can use to update the current URL. Just
-pass it a string like `/foo/bar?param=5`.
+An action creator that you can use to update the current URL and update the
+browser history. Just pass it a string like `/foo/bar?param=5` as the
+`path` argument.
+
+You can optionally pass a state to this action creator to update the state of the
+current route.
 
 The `noRouterUpdate`, if `true`, will stop react-router from reacting
```
Comment (kjbekkelund) [inline README.md]: This has been renamed

File: README.md [hunk 4]
```diff
@@ -154,15 +154,27 @@ piece of state must be named `routing`** (unless you provide a custom
...
 to this and all future URL changes. Pass `false` to make it start
```
Comment (kjbekkelund) [inline README.md]: We should remove "and all future" and the next sentence ("Pass `false` to make it start..."). `avoidRouterUpdate` is only for the specific action, not for later actions.

**Comment by ellbee** [comment]
Very important: `Just look at this code: the whole thing is only 68 lines of JS.` We need a recount here :wink:

**Comment by ellbee** [comment]
`The piece of state is just a URL string, whereas redux-router stores the full location object from react-router.`

The state is a little bit more complicated now. Maybe this should say something like:

`The piece of state of simpler. redux-router stores the entire location object from react-router`

**Comment by justingreenberg** [comment]
@geowarin please see #89 #93 i'm sorry i had included the README updates in previous PR and was asked to split out, had no idea you already tackled this

@ellbee i had reworded it to "We only store current URL and state, whereas redux-router stores the entire location object from react-router." Once we support search and query keys we can say we support the whole object. In reality this should be history, not react-router, but I left it as-is for simplicity

**Comment by geowarin** [comment]
@justingreenberg No problem, #93 is a much better PR. Should I close this one?
Maybe you could add the changelog to your PR also?

**Comment by justingreenberg** [comment]
@geowarin :smile: i added the changelog stuff to #93—thank you again! this PR be closed

**Pull request closed without merging**